trymito.io/pages/blog/[slug].tsx

```diff
@@ -1,88 +1,171 @@
-            </div>
-              <div className={postStyles.post_content}>
-                <div dangerouslySetInnerHTML={{ __html: props.post.html }} />
+      {/* All blog post related content */}
+      <main className={classNames(pageStyles.main, 'blog-content')}>
+        <div className={postStyles.post}>
+          <div className={postStyles.decorative_line}/>
+
+          {/* Blog Title Banner */}
+          <div className={postStyles.post_banner}>
+            {/* We use a wrapper around the post title to get the positioning in line with the post content. */}
+            <div className={postStyles.post_title}>
+              <div className={postStyles.blog_home_container}>
+                <Link href='/blog'>
+                  <div className={postStyles.blog_home_image_and_text}>
+                    <Image src={'/excel-to-python/house.svg'} alt='Home Icon' width={20} height={14} />
+                    <p>See more blogs</p>
+                  </div>
+                </Link>
+              </div>
+              <h1>{props.post.title}</h1>
+              <div className={postStyles.post_metadata}>
+                <p>
+                  { authorName ? 'By '+ props.post.primary_author?.name: 'The Mito Team' }
+                </p>
+                {publishedAt && <p className={postStyles.post_additional_info}> {publishedAt}</p>}
+                {readingTime && <p className={postStyles.post_additional_info}>{readingTime} min read</p>}
               </div>
             </div>
+          </div>
 
-            <section className={pageStyles.background_card}>
-              <DownloadCTACard />
-            </section>
-          </main>
+          {/* Table of Contents, post contents, and CTA */}
+          <div className={postStyles.post_content_container}>
+            {/* Table of Contents */}
+            <div className={postStyles.post_toc}>
+              <PageTOC />
+            </div>
+            {/* Post Contents */}
+            <div className={postStyles.post_content}> 
+              <div dangerouslySetInnerHTML={{ __html: props.post.html }}/>
+              <div className={postStyles.share_section}>
+                <a className={postStyles.tweet_button}
+                  href={`https://twitter.com/intent/tweet?text=${encodeURIComponent(props.post.title ?? '')}&url=${encodeURIComponent(currentURL ?? '')}`}>
+                  <TwitterLogo />
+                </a>
+                <a className={postStyles.email_button}
+                  href={`mailto:?subject=${encodeURIComponent(props.post.title ?? '')}&body=${encodeURIComponent(currentURL ?? '')}`}>
+                    <EmailIcon />
+                  </a>
+                <a className={postStyles.linkedin_button}
+                  href={`https://linkedin.com/shareArticle?title=${encodeURIComponent(props.post.title ?? '')}&url=${encodeURIComponent(currentURL ?? '')}`}>
+                    <LinkedinLogo />
```

Comment:
This button isn't working for me: 

<img width="563" alt="Screenshot 2024-02-19 at 10 37 43 AM" src="https://github.com/mito-ds/mito/assets/18709905/d8ae6a55-4da9-4730-9958-87c6c49dd421">

Reply:
Ah yeah that's not going to work locally because it uses the "current" url - I tested it with vercel and it worked (but used the vercel url)

Reply:
Ooo make sense!